docs/api/v2/node.md

```diff
@@ -8,20 +8,54 @@ TBD.
 
 *NOTE: Some of these are currently at the protocol layer rather than than the Node itself.*
 
-```
+### WakuSub API
+
+```Nim
 method publish*(w: WakuSub, topic: string, data: seq[byte]) {.async.}
 method subscribe*(w: WakuSub, topic: string, handler: TopicHandler) {.async.}
+method unsubscribe*(w: WakuSub, topics: seq[TopicPair]) {.async.}
+```
+
+### Waku API
+
+```Nim
+proc publish(w: WakuNode, topic: WakuTopic, data: seq[byte]): bool
+proc subscribe(w: WakuNode, topics: seq[string], handler: WakuHandler): Identifier
+proc unsubscribe(id: Identifier): bool
+# Can get rid of the identifiers here and do something like WakuSub.
```

Comment:
Well, because you could have a different handler but the same topics. Hence the identifier, but we could do the same as in the WakuSub API that you use the topics + handler.